webapp/src/js/enketo/widgets/db-object-widget.js

```diff
@@ -75,7 +87,7 @@ const changeHandler = function() {
   const doc = selected && selected[0] && selected[0].doc;
   if (doc) {
     const field = $this.attr('name');
-    const index = $('[name="' + field + '"]').index(this);
+    const index = $('select[name="' + field + '"]').index(this);
```

Comment:
So we end up having one hidden input and a select with the same name? 
Is that required?

Reply:
Yes that is where things sit now.  

> Is that required?

It is not required that they have the same name. Things go bad in the Enketo logic if I change the name on the `input` (which is to be expected). However, I do not have to set the _same_ name onto the `select`.  I do have to set some kind of name on the `select` or other weird behavior happens in Enekto (a `select` with no name confuses the Enekto node resolution logic in ways that I do not fully understand...).  But as far as I can tell it works fine to just set any random value as the name on the `select`.  I am currently just copying the name from the `input`, (and am not really seeing any adverse effects), but let me know if you think it would be better to set a static name for all the db-object `select`s (or to create some kind of dynamic name for each...).